execution/executor/tests/storage_integration_test.rs

```diff
@@ -376,9 +389,18 @@ fn test_extend_whitelist() {
         .account_keypair
         .as_mut()
         .unwrap();
-
     let validator_privkey = keys.take_private().unwrap();
     let validator_pubkey = keys.public_key();
+    let signer = ValidatorSigner::new(
+        config.validator_network.unwrap().peer_id,
+        config
```

Comment:
dude

Reply:
lol

Reply:
Oh the great will power it takes to code all these repetation!